packages/components/src/query-controls/index.js

```diff
@@ -37,6 +36,16 @@ export default function QueryControls( {
 	onOrderChange,
 	onOrderByChange,
 } ) {
+	const categorySuggestions = categoriesList.reduce(
+		( accumulator, category ) => ( {
+			...accumulator,
+			[ category.name ]: category,
+		} ),
+		{}
+	);
+
+	const MAX_CATEGORIES_SUGGESTIONS = 20;
```

Comment:
Great! Nitpick: can we move this constant next to the constants at the top of this file:

```
const DEFAULT_MIN_ITEMS = 1;
const DEFAULT_MAX_ITEMS = 100;
const MAX_CATEGORIES_SUGGESTIONS = 20;
```